app/services/ping.rb

```diff
@@ -47,6 +48,35 @@ def ping_database
       }
     end
 
+    def ping_cache
+      response = {
+        type: Rails.application.config.cache_store[0]
```

Comment:
I wonder if we actually want to leak this information.

Reply:
I was torn, I was trying to ensure that `cache` was always present (even when file type) and trying to come up with something "common" to put here.

Reply:
I could do:

For file:
```
  cache: {
    status:
    duration:
  }
```

For redis:
```
  cache: {
    status:
    duration_ms:
    servers: {
      "id": {
         status:
         duration_ms:
    }
  }
```

Reply:
What does the global level mean? Does status OK mean at least one server works or that all servers work? Same for duration: what is the value if there are multiple servers? Average? min? max?

Reply:
I thought you could set a cache item to some random value and read it back. You can time that and use that as a duration and global status. Then individual servers can provide additional info. At the end it does show if the system can still use a cache, which is what matters I guess.

But I don't know how Rails behaves with multiple servers. Does it pick one at random?

Reading the docs I think https://guides.rubyonrails.org/caching_with_rails.html#connection-pool-options might actually be good to address as well. Cache could become a bottleneck. https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-rediscachestore is also interesting (redis vs hiredis gem). And https://redis.io/docs/reference/eviction/ is also suggested as reading material. Perhaps that's something we should do before making it the default.